### DISCUSSION TITLE
Download solid_dmft using EasyBuild?

### DISCUSSION THREAD

**Discussion by TommyHenz** [TRIQS/solid_dmft#39] [Q&A]
Hi, this might be just a yes/no answer, but would it be possible to have solid_dmft available to download using EasyBuild? 
Due to my workplace restrictions, I cannot install a package using pip, cmake or docker, which is the only two options made available on the website. I have installed triqs, dft_tools, chytb and now I just wanted to install solid_dmft.

Regards,

**Comment by the-hampel** [comment]
Hi, 
right now there are no plans to add solid_dmft to easybuild. Since solid_dmft contains only python code (no compiling needed) the organizational overhead is too large as of now to add this to the easybuild repository.  We can think about a custom easybuild script in the future. For now I wonder why you can't use pip with explicitly pointing to the install directory, i.e.: 
```
pip3 install --prefix $TRIQS_ROOT solid_dmft
```
if the triqs module sets the $TRIQS_ROOT environment variable. Or is pip simply not available? 

Why would cmake not be an option? You do not need any sudo rights to install into the easybuild directory?

**Reply by TommyHenz** [reply]
Hello, the problem was that in my workstation machine pip and cmake are not installed and would require me to have sudo rights to install that; asked this question since my manager is off until mid january so I thought to install it via easybuild if it exists. No worries, I'll just wait and install it once I have pip or cmake installed!

**Reply by the-hampel** [reply]
Hm I see. In any case you can always just copy the folder `python/solid_dmft` to the installation directory of triqs python packages: 

```
$TRIQS_ROOT/lib/python3.10/site-packages
```

This is basically what pip is doing during the install process. Additionally you have to copy https://github.com/TRIQS/solid_dmft/blob/3.1.x/bin/solid_dmft.py into the bin dir of triqs 

```
$TRIQS_ROOT/bin
```
rename it to `solid_dmft` and make it executable with chmod. That are all steps for installing it. 

Hope that helps.
